modules/claims_api/config/schemas/v2/526.json

```diff
@@ -235,7 +235,7 @@
             "beginDate": {
               "description": "Begin date for the Veteran's new address.",
               "pattern": "^(?:[0-9]{4})-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2][0-9]|3[0-1])$",
-              "type": "string",
+              "type": ["string", "null"],
```

Comment:
The "endDate" field has `"nullable": true` and shows that in the documentation for the field. Need to add that to "beginDate" so that the documentation displays correctly.

Reply:
Good catch, [87d6640](https://github.com/department-of-veterans-affairs/vets-api/pull/18413/commits/87d6640d1c73327e7c24a6b9c3ebce02dc0d2f01)